src/components/HairCheck/HairCheck.css

```diff
@@ -0,0 +1,52 @@
+.hair-check {
+  background: white;
+  border: 1px solid grey;
+  padding: 1rem;
+  border-radius: 4px;
+  max-width: 600px;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+  justify-content: center;
+}
+
+.hair-check video {
+  transform: rotate3d(0, 1, 0, 180deg);
```

Comment:
This mirrors the video! You're right that in production, it turns into `rotateY`, which is... strange. When you disable it, it should "unmirror" your self-view. 

Can you see the `-webkit-transform: rotateY(180deg);` property in the live demo as well? You need to turn off both, but I wonder if it's different for Windows machines.

<img width="325" alt="Screenshot 2022-05-13 at 14 48 28" src="https://user-images.githubusercontent.com/12814720/168286372-83ca0f4f-9b6c-46c2-9506-a8c1b7bc5c1a.png">

In case you're wondering why I'm not using `transform: scaleX(-1);` (which would flip the video horizontally, producing the same effect) - I was going to add an animation to the video, which looks a lot better when you're using `rotate()` - [this blog post has a good example](https://pqina.nl/blog/flipping-images-with-css-and-javascript/). I